src/NuGet.Clients/NuGet.VisualStudio.Client/Properties/AssemblyInfo.cs

```diff
@@ -0,0 +1,9 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using System.Reflection;
+using System.Runtime.InteropServices;
+
+[assembly: AssemblyTitle("NuGet.VisualStudio.Client")]
```

Comment:
Actually this whole file is not needed. The assembly produced by the build is not included.